executor/src/witgen/jit/function_cache.rs

```diff
@@ -194,15 +194,22 @@ impl<'a, T: FieldElement> FunctionCache<'a, T> {
             .filter_map(|(i, b)| if b { Some(Variable::Param(i)) } else { None })
             .collect::<Vec<_>>();
 
-        log::trace!("Compiling effects...");
-        let function = compile_effects(
-            self.fixed_data.analyzed,
-            self.column_layout.clone(),
-            &known_inputs,
-            &code,
-            prover_functions,
-        )
-        .unwrap();
+        let compiled_jit = !matches!(std::env::var("POWDR_JIT_INTERPRETER"), Ok(val) if val == "1");
```

Comment:
As you know, I'm not so happy about changing the behaviour of a function by something that is not an argument to the function.

How do you use this environment variable?

Reply:
this seemed like the smallest change to have a way of enabling the interpreter, at least for now:
interpreter is disabled by default, and only enabled when `POWDR_JIT_INTERPRETER=1`

Reply:
You could enable it if the field is not supported by the compiler, for example. Then we test it on a lot of inputs and in the future, we can think about enabling it if the degree is small.